pkgs/applications/video/kodi/build-kodi-addon.nix

```diff
@@ -0,0 +1,21 @@
+{ stdenv, toKodiAddon, addonDir }:
+{ name ? "${attrs.pname}-${attrs.version}"
+, namespace
+, sourceDir ? ""
+, ... } @ attrs:
+toKodiAddon (stdenv.mkDerivation ({
+  name = "kodi-" + name;
+
+  dontStrip = true;
+
+  extraRuntimeDependencies = [ ];
```

Comment:
Empty ones should probably not be declared.

Reply:
From what I can gather that was there to avoid a check for non existent property. I believe this may have been fixed.... but will leave that as is for now. I think I can clean this up in a future PR.